web_src/js/features/contextPopup.js

```diff
@@ -0,0 +1,66 @@
+export default function initContextPopups(suburl) {
+  const refIssues = $('.ref-issue');
+  if (!refIssues.length) return;
+
+  refIssues.each(function () {
+    const [index, _issues, repo, owner] = $(this).attr('href').split('/').reverse();
```

Comment:
```suggestion
    const [index, _issues, repo, owner] = $(this).attr('href').replace(/[#?].*$/, '').split('/').reverse();
```

Reply:
Cut off query string and hash so `index` does not contain them. Ideally this would be done using `URL` constructor but that would require a additional polyfill for IE 11 😢

Reply:
https://regex101.com/r/2KIEFK/1/